presto-docs/src/main/sphinx/functions/binary.rst

```diff
@@ -61,3 +61,7 @@ Binary Functions
 .. function:: xxhash64(binary) -> varbinary
 
     Computes the xxhash64 hash of ``binary``.
+
+.. function:: crc32(binary) -> varbinary
+
+    Computes the crc32 hash of ``binary``.
```

Comment:
Updated the documentation, both returning a bigint and also encouraging people to use xxhash64.